packages/@aws-cdk/aws-appmesh/lib/virtual-node-listener.ts

```diff
@@ -30,6 +31,20 @@ interface VirtualNodeListenerCommonOptions {
    * @default - no healthcheck
    */
   readonly healthCheck?: HealthCheck;
+
+  /**
+   * The TLS mode.
+   *
+   * @default - none
+   */
+  readonly tlsMode?: TlsMode;
```

Comment:
Same question here, as in `virtual-gateway-listener.ts` (why have 2 places where you can set `TlsMode`?).